client/my-sites/plugins/with-plugin-redirect.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useRef, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useTranslate, translate as __ } from 'i18n-calypso';
```

Comment:
Is there a good reason why we alias `translate` to `__` here? I think it can be confusing and someone could think we're using `@wordpress/i18n` instead.

Reply:
in HOC, I created the `translate` form which was triggering a lint error. Let me update it.